src/icp_canister_backend/src/lib.rs

```diff
@@ -9,11 +9,13 @@ use std::cell::RefCell;
 lazy_static! {
     pub static ref TRANSFER_FEE: Nat = Nat::from(10_000u64);
     pub static ref MINIMUM_DEPOSIT_AMOUNT: Nat = Nat::from(100_000u64);
+    pub static ref EPISODE_DURATION: u64 = 91 * 24 * 60 * 60 / 3;
```

Comment:
This kind of constants doesn't require code execution, so put them outside lazy_static, and make them usual rust constants.